tests/playwright/src/utility/kubernetes.ts

```diff
@@ -0,0 +1,154 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Page } from '@playwright/test';
+import test, { expect as playExpect } from '@playwright/test';
+
+import type { KubernetesResourceState } from '../model/core/states';
+import type { KubernetesResources, PlayKubernetesOptions } from '../model/core/types';
+import { ContainerDetailsPage } from '../model/pages/container-details-page';
+import type { PodsPage } from '../model/pages/pods-page';
+import { NavigationBar } from '../model/workbench/navigation';
+import { handleConfirmationDialog } from './operations';
+
+export async function deployPodToCluster(
```

Comment:
You can deploy pod or container. So two pages, two options. Here you are saying deploy pod but then  ou are looking at containers page. It is confusing.